lib/paperclip/storage/fog.rb

```diff
@@ -172,6 +184,18 @@ def copy_to_local_file(style, local_dest_path)
 
       private
 
+      def url_protocol
+        if @options[:fog_host]
+          @options[:fog_host] =~ /https:\/\// ? 'https' : 'http'
+        else
+          if fog_credentials[:provider] == 'AWS'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.